.github/workflows/integration-tests-on-release.yaml

```diff
@@ -14,7 +14,6 @@ defaults:
 
 jobs:
   TestCephFlexSuite:
-    if: github.ref == 'refs/heads/master' || github.ref == 'refs/tags/release-*'
```

Comment:
@subhamkrai @leseb Do we need this condition for anything? This has already been merged to release-1.7 to unblock the builds, but before merging to master wanted to confirm if i'm missing something.

Reply:
right, we can remove it as we are already running this on
```
on:
  push:
    tags:
      - v*
    branches:
      - master
      - release-* 
  ```